### PR TITLE
[8579] Finis elkjop doc

### PR DESCRIPTION
**Pull request by DavidVergnaultMoank**

### CODE REVIEW

File: specifications.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@ paths:
         required: true
         schema: {}
       responses:
-        200:
+        204:
```
Comment (Erik-Nilsson-Fimento) [inline specifications.yml]: 204 😳

Comment (Erik-Nilsson-Fimento) [reply specifications.yml]: Based on what google tells me, even an empty object counts as 200, 204 is if there is no response at all

File: specifications.yml [hunk 2]
```diff
@@ -19,11 +19,52 @@ servers:
...
+  /application/customer:
+    post:
+      tags:
+      - Application endpoints
+      summary: Create a customer
+      description: Create a customer and return its credit information
+      requestBody:
+        $ref: "#/components/requestBodies/customerApplication"
+      responses:
+        200:
```
Comment (Erik-Nilsson-Fimento) [inline specifications.yml:31]: No status codes except for 200? 🤔

**Review by Erik-Nilsson-Fimento — CHANGES_REQUESTED** [review]

**Comment by Erik-Nilsson-Fimento** [comment]
We need the assignedCreditInfo endpoint aswell, otherwise the merchant will not be able to see the customers "köputrymme"

File: specifications.yml (hunk 1 above)
Comment (DavidVergnaultMoank) [reply specifications.yml]: res.json({}) would be a 200 but res.end() would be a 204 for me I think :thinking:

File: specifications.yml (hunk 2 above)
Comment (DavidVergnaultMoank) [reply specifications.yml:31]: still a wip :D

**Comment by DavidVergnaultMoank** [comment]
> We need the assignedCreditInfo endpoint aswell, otherwise the merchant will not be able to see the customers "köputrymme"

We can return that info in the customer get endpoint perhaps ?

**Review by Erik-Nilsson-Fimento — APPROVED** [review]

**Pull request merged**